wp-content/themes/bpr2018/resources/templates/partials/post/col-block-small.tpl.php

```diff
@@ -17,15 +17,14 @@ class="img-10"
           <span role="img" aria-label="<?php echo $pic_alt; ?>"> </span>
         </div>
       </div>
+      <div class="post-title-small">
+        <span itemprop="headline"><?php the_title(); ?></span>
+      </div>
     </a>
   </div>
 
   <div class="text col-sm-7">
-    <div class="post-title-small">
-      <a itemprop="url" href="<?php echo esc_url(get_permalink()); ?>">
-      <span itemprop="headline"><?php the_title(); ?></span>
-      </a>
-    </div>
+    
```

Comment:
Could you actually undo the changes to this file? I think this kind of change would actually break things if we were to ever use this template again.